[2024-06-03T17:38:18.218+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-03T17:38:18.234+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2019-06-01T00:00:00+00:00 [queued]>
[2024-06-03T17:38:18.240+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2019-06-01T00:00:00+00:00 [queued]>
[2024-06-03T17:38:18.240+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-03T17:38:18.251+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): blunders_etl> on 2019-06-01 00:00:00+00:00
[2024-06-03T17:38:18.256+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=97) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-03T17:38:18.257+0000] {standard_task_runner.py:63} INFO - Started process 101 to run task
[2024-06-03T17:38:18.257+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'chess_etl', 'blunders_etl', 'scheduled__2019-06-01T00:00:00+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/chess_dag.py', '--cfg-path', '/tmp/tmp6fcr70s0']
[2024-06-03T17:38:18.258+0000] {standard_task_runner.py:91} INFO - Job 17: Subtask blunders_etl
[2024-06-03T17:38:18.291+0000] {task_command.py:426} INFO - Running <TaskInstance: chess_etl.blunders_etl scheduled__2019-06-01T00:00:00+00:00 [running]> on host a5a4421409ea
[2024-06-03T17:38:18.348+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='chess_etl' AIRFLOW_CTX_TASK_ID='blunders_etl' AIRFLOW_CTX_EXECUTION_DATE='2019-06-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2019-06-01T00:00:00+00:00'
[2024-06-03T17:38:18.349+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-03T17:38:18.350+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2024-06-03T17:38:18.356+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-03T17:38:18.362+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/chess_dag.py", line 23, in blunders_etl_task
    return get_monthly_blunders(execution_date)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/blunder_etl_code.py", line 89, in get_monthly_blunders
    connection = BaseHook.get_connection('chess_project_s3')
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/hooks/base.py", line 83, in get_connection
    conn = Connection.get_connection_from_secrets(conn_id)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/connection.py", line 519, in get_connection_from_secrets
    raise AirflowNotFoundException(f"The conn_id `{conn_id}` isn't defined")
airflow.exceptions.AirflowNotFoundException: The conn_id `chess_project_s3` isn't defined
[2024-06-03T17:38:18.368+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=chess_etl, task_id=blunders_etl, run_id=scheduled__2019-06-01T00:00:00+00:00, execution_date=20190601T000000, start_date=20240603T173818, end_date=20240603T173818
[2024-06-03T17:38:18.376+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 17 for task blunders_etl (The conn_id `chess_project_s3` isn't defined; 101)
[2024-06-03T17:38:18.392+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-06-03T17:38:18.404+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-03T17:38:18.405+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-06-03T17:44:45.712+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-03T17:44:45.726+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2019-06-01T00:00:00+00:00 [queued]>
[2024-06-03T17:44:45.731+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2019-06-01T00:00:00+00:00 [queued]>
[2024-06-03T17:44:45.732+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-03T17:44:45.741+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): blunders_etl> on 2019-06-01 00:00:00+00:00
[2024-06-03T17:44:45.746+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=127) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-03T17:44:45.747+0000] {standard_task_runner.py:63} INFO - Started process 129 to run task
[2024-06-03T17:44:45.748+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'chess_etl', 'blunders_etl', 'scheduled__2019-06-01T00:00:00+00:00', '--job-id', '75', '--raw', '--subdir', 'DAGS_FOLDER/chess_dag.py', '--cfg-path', '/tmp/tmpczy4p3ck']
[2024-06-03T17:44:45.749+0000] {standard_task_runner.py:91} INFO - Job 75: Subtask blunders_etl
[2024-06-03T17:44:45.784+0000] {task_command.py:426} INFO - Running <TaskInstance: chess_etl.blunders_etl scheduled__2019-06-01T00:00:00+00:00 [running]> on host f55fff24b127
[2024-06-03T17:44:45.847+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='chess_etl' AIRFLOW_CTX_TASK_ID='blunders_etl' AIRFLOW_CTX_EXECUTION_DATE='2019-06-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2019-06-01T00:00:00+00:00'
[2024-06-03T17:44:45.848+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-03T17:44:45.849+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2024-06-03T17:44:45.855+0000] {base.py:84} INFO - Using connection ID 'chess_project_s3' for task execution.
[2024-06-03T17:44:46.928+0000] {logging_mixin.py:188} INFO - No games for period 2019_06
[2024-06-03T17:44:46.928+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-03T17:44:46.929+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-03T17:44:46.936+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=chess_etl, task_id=blunders_etl, run_id=scheduled__2019-06-01T00:00:00+00:00, execution_date=20190601T000000, start_date=20240603T174445, end_date=20240603T174446
[2024-06-03T17:44:46.963+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-03T17:44:46.977+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-03T17:44:46.978+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-06-08T14:10:13.955+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-08T14:10:13.970+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2019-06-01T00:00:00+00:00 [queued]>
[2024-06-08T14:10:13.975+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2019-06-01T00:00:00+00:00 [queued]>
[2024-06-08T14:10:13.975+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-08T14:10:13.985+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): blunders_etl> on 2019-06-01 00:00:00+00:00
[2024-06-08T14:10:13.990+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=136) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-08T14:10:13.991+0000] {standard_task_runner.py:63} INFO - Started process 138 to run task
[2024-06-08T14:10:13.991+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'chess_etl', 'blunders_etl', 'scheduled__2019-06-01T00:00:00+00:00', '--job-id', '215', '--raw', '--subdir', 'DAGS_FOLDER/chess_dag.py', '--cfg-path', '/tmp/tmp_aopif3m']
[2024-06-08T14:10:13.992+0000] {standard_task_runner.py:91} INFO - Job 215: Subtask blunders_etl
[2024-06-08T14:10:14.022+0000] {task_command.py:426} INFO - Running <TaskInstance: chess_etl.blunders_etl scheduled__2019-06-01T00:00:00+00:00 [running]> on host 3c9d89bfdaf1
[2024-06-08T14:10:14.086+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='chess_etl' AIRFLOW_CTX_TASK_ID='blunders_etl' AIRFLOW_CTX_EXECUTION_DATE='2019-06-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2019-06-01T00:00:00+00:00'
[2024-06-08T14:10:14.087+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-08T14:10:14.088+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2024-06-08T14:10:14.094+0000] {base.py:84} INFO - Using connection ID 'chess_project_s3' for task execution.
[2024-06-08T14:10:15.013+0000] {logging_mixin.py:188} INFO - No games for period 2019_06
[2024-06-08T14:10:15.014+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-08T14:10:15.014+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-08T14:10:15.022+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=chess_etl, task_id=blunders_etl, run_id=scheduled__2019-06-01T00:00:00+00:00, execution_date=20190601T000000, start_date=20240608T141013, end_date=20240608T141015
[2024-06-08T14:10:15.047+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-08T14:10:15.060+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-08T14:10:15.061+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-06-08T14:51:41.464+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-08T14:51:41.478+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2019-06-01T00:00:00+00:00 [queued]>
[2024-06-08T14:51:41.484+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2019-06-01T00:00:00+00:00 [queued]>
[2024-06-08T14:51:41.484+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-08T14:51:41.493+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): blunders_etl> on 2019-06-01 00:00:00+00:00
[2024-06-08T14:51:41.499+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=253) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-08T14:51:41.500+0000] {standard_task_runner.py:63} INFO - Started process 255 to run task
[2024-06-08T14:51:41.500+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'chess_etl', 'blunders_etl', 'scheduled__2019-06-01T00:00:00+00:00', '--job-id', '251', '--raw', '--subdir', 'DAGS_FOLDER/chess_dag.py', '--cfg-path', '/tmp/tmp4bp0v80t']
[2024-06-08T14:51:41.501+0000] {standard_task_runner.py:91} INFO - Job 251: Subtask blunders_etl
[2024-06-08T14:51:41.532+0000] {task_command.py:426} INFO - Running <TaskInstance: chess_etl.blunders_etl scheduled__2019-06-01T00:00:00+00:00 [running]> on host bf0ade909435
[2024-06-08T14:51:41.593+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='chess_etl' AIRFLOW_CTX_TASK_ID='blunders_etl' AIRFLOW_CTX_EXECUTION_DATE='2019-06-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2019-06-01T00:00:00+00:00'
[2024-06-08T14:51:41.595+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-08T14:51:41.596+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2024-06-08T14:51:41.602+0000] {base.py:84} INFO - Using connection ID 'chess_project_s3' for task execution.
[2024-06-08T14:51:42.516+0000] {logging_mixin.py:188} INFO - No games for period 2019_06
[2024-06-08T14:51:42.517+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-08T14:51:42.517+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-08T14:51:42.523+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=chess_etl, task_id=blunders_etl, run_id=scheduled__2019-06-01T00:00:00+00:00, execution_date=20190601T000000, start_date=20240608T145141, end_date=20240608T145142
[2024-06-08T14:51:42.556+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-08T14:51:42.567+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-08T14:51:42.568+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-07-07T21:20:15.902+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-07T21:20:15.919+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2019-06-01T00:00:00+00:00 [queued]>
[2024-07-07T21:20:15.925+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2019-06-01T00:00:00+00:00 [queued]>
[2024-07-07T21:20:15.926+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-07-07T21:20:15.935+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): blunders_etl> on 2019-06-01 00:00:00+00:00
[2024-07-07T21:20:15.941+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1282) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-07T21:20:15.943+0000] {standard_task_runner.py:63} INFO - Started process 1285 to run task
[2024-07-07T21:20:15.942+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'chess_etl', 'blunders_etl', 'scheduled__2019-06-01T00:00:00+00:00', '--job-id', '580', '--raw', '--subdir', 'DAGS_FOLDER/chess_dag.py', '--cfg-path', '/tmp/tmpq2hsp1ug']
[2024-07-07T21:20:15.943+0000] {standard_task_runner.py:91} INFO - Job 580: Subtask blunders_etl
[2024-07-07T21:20:15.979+0000] {task_command.py:426} INFO - Running <TaskInstance: chess_etl.blunders_etl scheduled__2019-06-01T00:00:00+00:00 [running]> on host 6569f5155f0c
[2024-07-07T21:20:16.037+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='chess_etl' AIRFLOW_CTX_TASK_ID='blunders_etl' AIRFLOW_CTX_EXECUTION_DATE='2019-06-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2019-06-01T00:00:00+00:00'
[2024-07-07T21:20:16.038+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-07T21:20:16.039+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2024-07-07T21:20:16.044+0000] {base.py:84} INFO - Using connection ID 'chess_project_s3' for task execution.
[2024-07-07T21:20:16.942+0000] {logging_mixin.py:188} INFO - No games for period 2019_06
[2024-07-07T21:20:16.943+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-07-07T21:20:16.943+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-07T21:20:16.950+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=chess_etl, task_id=blunders_etl, run_id=scheduled__2019-06-01T00:00:00+00:00, execution_date=20190601T000000, start_date=20240707T212015, end_date=20240707T212016
[2024-07-07T21:20:16.999+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-07T21:20:17.017+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-07-07T21:20:17.018+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
